chain/chain/src/store_validator.rs

```diff
@@ -69,7 +71,7 @@ impl StoreValidator {
                 format!("{:?}", CryptoHash::try_from(key.as_ref()))
             }
             DBCol::ColChunks => format!("{:?}", ChunkHash::try_from_slice(key.as_ref())),
-            _ => format!("{:?}", key),
+            _ => format!("{:?}", std::str::from_utf8(key)),
```

Comment:
hexify or base58 it instead? Converting a hash to utf8 is just weird :)

Reply:
it's only for manually set keys
there should be deserialization for known columns